### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards-Designer tab-New card]: The 'Create' dialog does not adapt high contrast aquatic mode.

### ISSUE THREAD

**Issue by HasnuddinMohammed** [microsoft/AdaptiveCards#9157]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test environment:** 
OS: Win11 
OS Version: 23H2 (OS Build 22631.5189) 
URL: https://adaptivecards.io/ 
Edge dev Browser version: 137.0.3255.0 (Official build) dev (64-bit) 
Screen reader: Narrator. 

 **Pre requisites:** 
Turn on the High contrast mode form System settings > Accessibility > contrast theme   

**Repro Steps:** 
1.Open URL in the Edge dev browser & sign in with valid credentials 
2.Navigate and activate the Designer tab. 
3.Activate the new card option observe the issue.

**Actual Result:** 
The background of 'Create' dialog does not change to black while using high contrast aquatic mode. 

**Expected Result:** 
The background of 'Create' dialog should change to black while using high contrast aquatic mode. 

**User Experience:** 
Low vision users depends on the high contrast modes unable to see the controls on the page if The background of 'Create' dialog does not change to black while using high contrast aquatic mode. 

**MAS Reference:** 
[MAS 4.3.1 – No Disruption of Accessibility Features](https://aka.ms/MAS4.3.1)

### Screenshots

<img width="1831" alt="Image" src="https://github.com/user-attachments/assets/a943ded4-f9b0-4485-bf95-43ebf373eb19" />

<img width="950" alt="Image" src="https://github.com/user-attachments/assets/1426237b-0a0e-4af4-bde2-318c49ed27e9" />

### Card JSON

```JSON
NA
```
